nni/retiarii/nn/pytorch/nn.py

```diff
@@ -29,12 +32,18 @@
     'ConstantPad3d', 'Bilinear', 'CosineSimilarity', 'Unfold', 'Fold',
     'AdaptiveLogSoftmaxWithLoss', 'TransformerEncoder', 'TransformerDecoder',
     'TransformerEncoderLayer', 'TransformerDecoderLayer', 'Transformer',
-    #'LazyLinear', 'LazyConv1d', 'LazyConv2d', 'LazyConv3d',
-    #'LazyConvTranspose1d', 'LazyConvTranspose2d', 'LazyConvTranspose3d',
-    #'Unflatten', 'SiLU', 'TripletMarginWithDistanceLoss', 'ChannelShuffle',
-    'Flatten', 'Hardsigmoid', 'Hardswish'
+    'Flatten', 'Hardsigmoid'
 ]
 
+if version.parse(torch.__version__) >= version.parse('1.6.0'):
```

Comment:
Is this "the" correct checking? I saw many occurrences in our repo where:

![image](https://user-images.githubusercontent.com/8463288/102474583-c8502d80-4093-11eb-8975-63ef36caa2a3.png)